### ISSUE TITLE
Opposite view between norm and quality CASIA-Webface

### ISSUE THREAD

**Issue by NNDam** [mk-minchul/AdaFace#49]
Hi, thanks for yours excellent work! It help me a lot.
I've add your AdaFace loss with [PartialFC](https://github.com/deepinsight/insightface/tree/master/recognition/arcface_torch) to deal with OOM problem when training (ultra) large scale dataset and get some questions:
- I've analyze our Cosface model trained with PartialFC and get opposite view between norm & quality
![image](https://user-images.githubusercontent.com/40813954/185360694-6d3e66a9-21f4-4119-8e14-30a371ee7c24.png)
- My [implemented](https://github.com/NNDam/adaface-partialfc) also have a opposite with small CASIA-Webface dataset (10k ids, 0.5M images), but the accuracy for BLUR domain is still higher (I think the adaptive margin loss didn't work and the main improment of accuracy was only given by augmentation)
![image](https://user-images.githubusercontent.com/40813954/185363569-b56541c9-6137-4ebe-ad2b-21cfb3ac3fc4.png)
- But for my large scale training (5M ids, 105M images), your conclusion is still working
```
# Our current checkpoint
Training: 2022-08-18 08:10:35,869-[lfw][210000]XNorm: 23.302152
Training: 2022-08-18 08:10:35,870-[lfw][210000]Accuracy-Flip: 0.99800+-0.00296
Training: 2022-08-18 08:10:35,870-[lfw][210000]Accuracy-Highest: 0.99800

Training: 2022-08-18 08:11:33,067-[cfp_fp][210000]XNorm: 19.032604
Training: 2022-08-18 08:11:33,067-[cfp_fp][210000]Accuracy-Flip: 0.96914+-0.00833
Training: 2022-08-18 08:11:33,067-[cfp_fp][210000]Accuracy-Highest: 0.96914

Training: 2022-08-18 08:12:21,623-[agedb_30][210000]XNorm: 21.835075
Training: 2022-08-18 08:12:21,624-[agedb_30][210000]Accuracy-Flip: 0.97567+-0.00782
Training: 2022-08-18 08:12:21,624-[agedb_30][210000]Accuracy-Highest: 0.97733
Training: 2022-08-18 08:12:21,624-[+][210000]Score / Score-Highest: 2.94281 / 2.94362
```
Have we need to re-thinking about the properties of training dataset ?

**Comment by wuxuedaifu** [comment]
hello, can you share the training loss log for Adaface,, thank you
